### ISSUE TITLE
Auto-close import asset pop-up

### ISSUE THREAD

**Issue by nihal467** [coronasafe/care_fe#7607]
**Describe the bug**

When we complete the asset import, the pop-up is not auto closing

**To Reproduce**

1. Go to asset tab, click on the import asset icon
2. Upload a excel file
3. once the upload is complete

**Expected behavior**

Upon completing the upload, the import assets pop-up should auto-close

**Screenshots**

![image](https://github.com/coronasafe/care_fe/assets/57055998/5c93d6c6-399e-42eb-a252-7a07fc9c8872)

**Comment by Sulochan-khadka** [comment]
@nihal467 may I solve this issue? If and only if it is not already reported.

**Comment by Sulochan-khadka** [comment]
@nihal467 may you provide me the file that is accepted in this or instruct me to solve "Please check the row 2 of the column Last Service Date" error.

**Comment by nihal467** [comment]
@Sulochan-khadka just fill in the asset creation mandatory field in the excel sheet and try to upload it

**Comment by UdaySagar-Git** [comment]
@Sulochan-khadka @nihal467 this issue is already covered in my pr https://github.com/coronasafe/care_fe/pull/7367

i will fix the merge conficts and push it as soon as possible

**Comment by Sulochan-khadka** [comment]
@UdaySagar-Git it is great that you solved the issue. But hear me out once mate, solving multiple issues with a single PR is a great achievement but solving issues in a PR that does not associate with it primarily is not a good idea. The reason is , tracking the issues and its corresponding PR becomes not very easy. I saw your PR and it was not originally meant to solve this issue. Please do not take my statements otherwise, just wanted to share some workflow tips that I gained from this repo.

**Comment by Sulochan-khadka** [comment]
BTW, thank you for solving the issue @UdaySagar-Git , @nihal467 just assign this issue to @UdaySagar-Git and mention this issue in his PR #7367

**Comment by UdaySagar-Git** [comment]
@Sulochan-khadka , I completely agree with you, buddy! 😀 That PR involved a couple of changes in packages, which in turn resulted in a very large PR. 
I'll keep that in mind from next time.
